src/VirtoCommerce.Platform.Caching/ServiceCollectionExtensions.cs

```diff
@@ -21,6 +22,10 @@ public static IServiceCollection AddCaching(this IServiceCollection services, IC
                 services.AddOptions<RedisCachingOptions>().Bind(configuration.GetSection("Caching:Redis")).ValidateDataAnnotations();
 
                 services.AddSingleton<IPlatformMemoryCache, RedisPlatformMemoryCache>();
+
```

Comment:
In the VC the Redis is used not only for caching purposes and also for PushNotifications. You can't move the redis connection only into Cache project.